project-words.txt

```diff
@@ -0,0 +1,64 @@
+Vfile
+Zenodo
+GOTBIN
+Omics
+NBIS
+Viwewer
```

Comment:
```suggestion
```
This word appears in `news.json`: " Other small tweaks on the <i>Map Viwewer</i> are:"